packages/carrot-scripts/src/scripts/start.ts

```diff
@@ -194,22 +194,25 @@ const main = async () => {
         secretKey: Hex,
         deploymentAccountInitialBalance: bigint;
     try {
-        const anvil = createAnvil({
-            silent: false,
-            forkUrl,
-            noStorageCaching: true,
-            forkChainId: forkedChain.id,
-            chainId: forkedChain.id,
-            forkBlockNumber: await forkPublicClient.getBlockNumber(),
-            mnemonic: MNEMONIC,
-            derivationPath: DERIVATION_PATH,
-            accounts: 1,
-            port: ANVIL_PORT,
-            host: "127.0.0.1",
+        const ganacheServer = ganache.server({
+            fork: { url: forkUrl, deleteCache: true, disableCache: true },
+            chain: {
+                chainId: forkedChain.id,
+            },
+            wallet: {
+                totalAccounts: 0,
```

Comment:
Shouldn't we have at least 1 account here?

Reply:
In the previous implementation we had it, but now we just generate an account from the mnemonic and derivation path through `viem` and set its balance manually. The end result is the same, but I was having some issues with Ganahce's typings so I opted to go down this road.